code/game/turfs/simulated.dm

```diff
@@ -17,7 +17,7 @@
 	if(wet >= wet_setting)
 		return
 	wet = wet_setting
-	if(wet_setting == TURF_WET_WATER)
+	if(wet_setting == TURF_WET_WATER || TURF_WET_LUBE)

```

Comment:
`if(wet_setting == TURF_WET_WATER || TURF_WET_LUBE)`
this checking two separate conditions; they are:
`wet_setting == TURF_WET_WATER` and `TURF_WET_LUBE`
`if(TURF_WET_LUBE)` will evaluate to true as long as it's a truthy value (anything other than 0, null, or "")